bug-bounty-hof/web.yml

```diff
@@ -1,4 +1,54 @@
 names:
+- name: endang
+  date: 2022-06-27
+- name: redparsec
+  date: 2022-06-13
+  twitter: "https://twitter.com/r3dpars3c"
+  url: https://twitter.com/https://twitter.com/r3dpars3c
```

Comment:
```suggestion
  url: https://twitter.com/r3dpars3c
```

Reply:
What Frida said about the `url:`, but also the `twitter:` value should be "@r3dpars3c" rather than the URL again.